compiled_starters/javascript/app/main.js

```diff
@@ -0,0 +1,15 @@
+// Uncomment this block to pass the first stage
+// const readline = require("readline");
+
+// You can use print statements as follows for debugging, they'll be visible when running tests.
+console.log("Logs from your program will appear here!");
+
+// Uncomment this block to pass the first stage
+// const rl = readline.createInterface({
```

Comment:
@glokta1 let's not use readline by default, just print using `console.log` (like we do for the starter code in other challenges).

Reasons: 

- We might make readline usage an extension in the future
- Not all users will be familiar with readline, we don't want to overwhelm them at this point

Reply:
We decided against this since there doesn't seem to be a clean 'gets'-like interface for node, and all code examples on the internet seem to suggest using `readline`.